### DISCUSSION TITLE
좋아요 기능 적용 범위 및 디자인 논의

### DISCUSSION THREAD

**Discussion by vi-wolhwa** [woowacourse-teams/2024-code-zap#671] [결정 사항]
좋아요 기능의 적용 범위 및 디자인에 대한 논의입니다.

### 1. 기능 적용 범위
- 템플릿 디테일
- 템플릿 카드
  - 내 템플릿
  - 구경가기

### 2. 디자인
좋아요 기능을 적용하는 각 요소에서의 디자인
- 좋아요를 어느 부분에 넣을 것인가?

**Comment by vi-wolhwa** [comment]
## 논의 결과

### 1. 기능 적용 범위
#### 1. 템플릿 디테일
  - 좋아요 수 확인 가능
  - 좋아요 클릭 가능 (내 템플릿 좋아요 가능)
#### 2. 템플릿 카드 (추후 논의 필요)
  - 좋아요 수 확인 가능
  - 좋아요 클릭 불가능

<hr>

### 2. 디자인
#### 1. 템플릿 디테일
  - v1. 메타데이터와 같은 라인에 좋아요 버튼 추가
  - v2. 메타데이터를 위로 올리고 같은 라인에 편집버튼, 제목과 같은 라인에 좋아요 버튼 추가
#### 3. 템플릿 카드
  - 작성자명 + 업데이트 일자, 같은 라인 우측에 좋아요 카운터 (내가 눌렀는지 색깔로 표시 검빨)

**Comment by Jaymyong66** [comment]
![스크린샷 2024-09-19 오후 5 39 55](https://github.com/user-attachments/assets/42f67982-00a4-4122-b145-5514ccd131b1)
